lib/trace_logger.cc

```diff
@@ -40,7 +40,11 @@ TraceLogger( uint8_t const level, char const * const file_name_format, ... )
 
     _stream_handle = fopen( tfile_name, "w" );
     if (NULL == _stream_handle) {
-        throw InvalidStreamBuffer( );
+        std::ostringstream msg;
```

Comment:
Nope, just got caught in my `git grep` marathon. The `InvalidStreamBuffer` exception class doesn't exist any longer.